src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+.page__body {
+  margin: 0;
+}
```

Comment:
It's a good practice to provide a fallback font-family in case the main one doesn't load. Consider adding a generic font family like 'sans-serif' as a fallback.